Component/CustomerGroups.php

```diff
@@ -60,9 +60,8 @@ public function execute($data = null)
             if ($taxClassId) {
                 foreach ($taxClass['groups'] as $group) {
                     try {
-                        if (isset($group['name'])) {
-                            $this->createCustomerGroup($group['name'], $taxClassId);
-                        }
+                        $this->validateGroupName($group);
```

Comment:
essentially, this change is so that when the group is not created, we can work out why without having to get into the code